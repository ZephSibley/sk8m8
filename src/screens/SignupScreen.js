import React from 'react';

import '../styles/screens.css';
import SignupForm from '../components/ui/forms/SignupForm';

const SignupScreen = () => {

    return (
        <div className={'form-screen'}>
            <h1>S K 8 M 8</h1>
            <SignupForm />
        </div>
    );
}

export default SignupScreen