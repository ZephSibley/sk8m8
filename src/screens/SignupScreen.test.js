import React from 'react';
import renderer from 'react-test-renderer';
import SignupScreen from './SignupScreen';

test('Basic Rendering', () => {
    const component = renderer.create(
        <SignupScreen />
    )
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot();
})