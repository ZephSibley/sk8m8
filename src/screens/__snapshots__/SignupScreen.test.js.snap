// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Basic Rendering 1`] = `
<div
  className="form-screen"
>
  <svg
    aria-hidden="true"
    className="MuiSvgIcon-root makeStyles-root-1"
    focusable="false"
    onClick={[Function]}
    role="presentation"
    viewBox="0 0 24 24"
  >
    <path
      d="M20 11H7.83l5.59-5.59L12 4l-8 8 8 8 1.41-1.41L7.83 13H20v-2z"
    />
  </svg>
  <h1>
    S K 8 M 8
  </h1>
  <div>
    <form
      className="makeStyles-form-11"
      onSubmit={[Function]}
    >
      <div
        className="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
          data-shrink={false}
        >
          email
        </label>
        <div
          className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
          onClick={[Function]}
        >
          <input
            aria-invalid={false}
            className="MuiInputBase-input MuiInput-input"
            disabled={false}
            name="email"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            required={false}
            type="email"
            value=""
          />
        </div>
        
      </div>
      <div
        className="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
          data-shrink={false}
        >
          confirm email
        </label>
        <div
          className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
          onClick={[Function]}
        >
          <input
            aria-invalid={false}
            className="MuiInputBase-input MuiInput-input"
            disabled={false}
            name="confirm_email"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            required={false}
            type="email"
            value=""
          />
        </div>
        
      </div>
      <div
        className="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
          data-shrink={false}
        >
          username
        </label>
        <div
          className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
          onClick={[Function]}
        >
          <input
            aria-invalid={false}
            className="MuiInputBase-input MuiInput-input"
            disabled={false}
            name="username"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            required={false}
            type="username"
            value=""
          />
        </div>
        
      </div>
      <div
        className="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
          data-shrink={false}
        >
          password
        </label>
        <div
          className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
          onClick={[Function]}
        >
          <input
            aria-invalid={false}
            className="MuiInputBase-input MuiInput-input"
            disabled={false}
            name="password"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            required={false}
            type="password"
            value=""
          />
        </div>
        
      </div>
      <div
        className="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
          data-shrink={false}
        >
          confirm password
        </label>
        <div
          className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
          onClick={[Function]}
        >
          <input
            aria-invalid={false}
            className="MuiInputBase-input MuiInput-input"
            disabled={false}
            name="confirm_password"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            required={false}
            type="password"
            value=""
          />
        </div>
        
      </div>
      <span
        className="MuiFormHelperText-root Mui-error"
      />
      <button
        className="MuiButtonBase-root MuiButton-root makeStyles-submit-15 MuiButton-contained MuiButton-containedPrimary Mui-disabled Mui-disabled"
        disabled={true}
        margin="normal"
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={-1}
        type="submit"
      >
        <span
          className="MuiButton-label"
        >
          Sign up
        </span>
      </button>
    </form>
  </div>
  <div
    style={
      Object {
        "marginBottom": 100,
      }
    }
  />
</div>
`;
